File: Personagens/Lucius/lucius.c
#include "lucius.h"

void iniciarLucius(Lucius *lucius) {
    lucius->vidaMaxima = 300;
    lucius->vida = lucius->vidaMaxima;

    lucius->forca = 22;

    lucius->resistenciaBase = 20;
    lucius->bonusExoesqueleto = 40;
    lucius->resistencia =
        lucius->resistenciaBase +
        (lucius->resistenciaBase * lucius->bonusExoesqueleto / 100);

    lucius->evasao = 12;
    lucius->sangramento = 0;
}

File: personagens/Lucius.java
package personagens;

public class Lucius {
    public int vida;
    public int vidaMaxima;
    public int forca;
    public int resistenciaBase;
    public int resistencia;
    public int evasao;

    public int sangramento;
    public int turnosSangramento;
    public int bonusExoesqueleto;

    public Lucius() {
        vidaMaxima = 300;
        vida = vidaMaxima;
        forca = 22;

        resistenciaBase = 20;
        bonusExoesqueleto = 40;
        resistencia = resistenciaBase + (resistenciaBase * bonusExoesqueleto / 100);

        evasao = 12;
        sangramento = 0;
        turnosSangramento = 0;
    }
}
